wire-schema/src/test/java/com/squareup/wire/schema/SchemaTest.kt

```diff
@@ -399,7 +399,7 @@ class SchemaTest {
   }
 
   @Test
-  fun oneofFieldTypeUnknown() {
+  fun oneOfFieldTypeUnknown() {
```

Comment:
Here the method name is not in proper lower camel case unless if we consider "oneof" as a single word(I have some proofs please see below).
In some part of the class, the word("oneof") is used in perfect lower camel case(`fun oneOfGroupsThrow()`,`fun duplicateTagValueDisallowedInOneOf()`). Incase these changes are irrelevant I will change the pull request.